src/locale/locales/fr/messages.po

```diff
@@ -186,13 +187,12 @@ msgstr "Mots de passe de l’appli"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:236
 msgid "Appeal content warning"
-msgstr ""
+msgstr "Avertissement sur le contenu de l'appel"
 
 #: src/view/com/modals/AppealLabel.tsx:65
 msgid "Appeal Content Warning"
-msgstr ""
+msgstr "Avertissement sur le Contenu de L'appel"
```

Comment:
This translation is OK, but let's note that we don't do Title Casing in UI in software localization in French. I would recommand keeping it like `Avertissement sur le contenu de l'appel`, without special casing applied (beside the first letter, of course).

Reply:
Done